Makefile

```diff
@@ -173,6 +173,8 @@ $(EXES): loki-build-image/$(UPTODATE)
 
 lint: loki-build-image/$(UPTODATE)
 	GOGC=20 golangci-lint run
+	@git diff --name-only HEAD master | xargs --no-run-if-empty -n1 sed -i "s#[ \t]*$##"
```

Comment:
can you explain what that does ?

Reply:
use `sed` to cleanup white noise, and then use `git diff` to check any diffenerces.